cincinnati/src/plugins/internal/cincinnati_graph_fetch.rs

```diff
@@ -14,16 +14,23 @@ use failure::{Fallible, ResultExt};
 use prometheus::Counter;
 use reqwest;
 use reqwest::header::{HeaderValue, ACCEPT};
+use std::time::Duration;
 
 /// Default URL to upstream graph provider.
 pub static DEFAULT_UPSTREAM_URL: &str = "http://localhost:8080/v1/graph";
 
+/// Default graph-builder connection timeout in seconds.
+pub static DEFAULT_GB_TIMEOUT_SECS: u64 = 30;
+
 /// Plugin settings.
 #[derive(Clone, CustomDebug, Deserialize, SmartDefault)]
 #[serde(default)]
 struct CincinnatiGraphFetchSettings {
     #[default(DEFAULT_UPSTREAM_URL.to_string())]
     upstream: String,
+
+    #[default(DEFAULT_GB_TIMEOUT_SECS)]
+    gb_timeout: u64,
```

Comment:
Sorry for the late nitpick, but the `gb_` prefix is not appropriate. The plugin fetches a graph, it could do so from any graph endpoint. Let's just call it timeout?

```suggestion
    timeout: u64,
```

Reply:
Yeah, sure